Cargo.toml

```diff
@@ -14,3 +14,4 @@ libc = "0.2"
 
 [features]
 no_wrapper = []
+static = []
```

Comment:
What about two features `libv4l2_static` + `libv4_dylib` and `default = ["libv4l2_dylib"]`?

Reply:
Mutually exclusive features aren't supported.